extensions/wikia/WidgetFramework/Widgets/WidgetProblemReports/WidgetProblemReports.php

```diff
@@ -133,7 +133,7 @@ function WidgetProblemReports($id, $params) {
 	    'href'  => $baseUrl.'/'.$problem['id'],
 	    'title' => '#'.$problem['id'].' - '.wfMsg('pr_table_date_submitted'). ': ' . $date,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)

[project-id]: wikia:app